src/syncAzure.proj

```diff
@@ -10,6 +10,9 @@
     <BlobNamePrefix>$(__BlobNamePrefix)</BlobNamePrefix>
     <DownloadDirectory Condition="'$(PublishTestNativeBins)' != 'true'">$(PackagesDir)AzureTransfer</DownloadDirectory>
     <DownloadDirectory Condition="'$(PublishTestNativeBins)' == 'true'">$(PackagesDir)TestNativeBins\$(RuntimeId)</DownloadDirectory>
+
+    <!-- We don't want to run sync on test native binaries for Windows, since those don't get published -->
+    <SyncingNativeOnWindows Condition="'$(PublishTestNativeBins)' == 'true' and $(RuntimeId.ToLower().StartsWith('win'))">true</SyncingNativeOnWindows>
```

Comment:
Is the intent of `SyncingNativeOnWindows` to allow a hook so that you could override its value and force the target below to evaluate, or is it just to capture the two condition cases specified?  If it's the former, then that makes sense.  If it's the latter (because overriding this isn't really valid), then I'd just move the condition into the target below.